app/pages/patient/patientinfo.js

```diff
@@ -399,6 +423,22 @@ var PatientInfo = React.createClass({
       return 'Diagnosis date needs to be a valid date';
     }
 
+    currentDateObj = currentDateObj || new Date();
+    var birthdayDateObj = sundial.parseFromFormat(birthday, FORM_DATE_FORMAT);
+    var diagnosisDateObj = sundial.parseFromFormat(diagnosisDate, FORM_DATE_FORMAT);
+
+    if (birthdayDateObj > currentDateObj) {
```

Comment:
Awesome, thanks! In the future we will have a timezone to pass through to sundial because we will be offering a UI for deciding what timezone to view data in (which will be defaulted to the timezone most recently selected from the uploader for a device upload), so we will have a bit of a better solution to this, but for now option 2 sounds good - LMK when you've updated. I'm looking at #234 now and we may need to coordinate updates if that's ready to merge.